astropy/time/formats.py

```diff
@@ -72,6 +73,11 @@ def _regexify_subfmts(subfmts):
     return tuple(new_subfmts)
 
 
+class TimeMaskedArray(np.ma.MaskedArray):
```

Comment:
Otherwise you get this:
```
In [9]: tm.cxcsec
Out[9]: 
masked_array(data = [99.99999999999805 -- 299.999999999999],
             mask = [False  True False],
       fill_value = 1e+20)
```
Instead of:
```
In [4]: tm.cxcsec
Out[4]: [99.99999999999805 -- 299.999999999999]
```